src/lib/Containers/Conversation.tsx

```diff
@@ -100,6 +100,8 @@ export class Conversation extends React.Component<Props, State> {
     return (
       <Message
         message={item}
+        firstMessage={item.first_message}
```

Comment:
the parent component is the one that 'knows' whether a message is the initial one or not, as well as its own initial text, so it passes them as props to the message component (used above)

Reply:
maybe later we can use some data in `metadata` to figure out if it's a inquiry initial message or not. for now this should be good too.

Reply:
Sure, or `order` maybe works as well. Currently this is assigned from the parent, to the first message received when querying `messages`.